src/components/job-definition-row.tsx

```diff
@@ -134,8 +117,8 @@ export function buildJobDefinitionRow(
           forceReload();
         }}
       />
-      <DeleteButton
-        jobDef={jobDef}
+      <DeleteWithConfirmationIcon
```

Comment:
```suggestion
      <ConfirmDeleteButton
```

Reply:
As described above, renamed `DeleteWithConfirmationButton` element into `ConfirmDeleteButton` and `DeleteWithConfirmationIcon` into `ConfirmDeleteIcon`